coldfront/core/user/forms.py

```diff
@@ -164,6 +176,38 @@ class UserProfileUpdateForm(forms.Form):
         help_text='The number must be in E.164 format (e.g. +12125552368).',
         label='Phone Number', required=False)
 
+    def clean_phone_number(self):
+        cleaned_data = super().clean()
+        phone_number = cleaned_data.pop('phone_number', '')
+        return phone_number
+
+    def clean_first_name(self):
+        cleaned_data = super().clean()
+        name = self.cleaned_data.pop('first_name', '')
+        return name.title()
+
+    def clean_middle_name(self):
+        cleaned_data = super().clean()
+        name = cleaned_data.pop('middle_name', '')
+        return name.title()
+
+    def clean_last_name(self):
+        cleaned_data = super().clean()
+        name = self.cleaned_data.pop('last_name', '')
+        return name.title()
+
+    def save(self, commit=True):
+        model = super().save(commit=False)
+        model.username = model.email
+        model.is_active = False
+        if commit:
+            model.save()
+        model.refresh_from_db()
+        model.userprofile.phone_number = self.phone_number
```

Comment:
The issue from above of these attributes not being previously set also applies here.